docs(sessions): Add docs for unhandled session status (#15086)
getsentry/sentry-docs: docs/product/releases/health/index.mdx

![Sort By: Active Users dropdown](./img/release_active_users.png)

> ## Crashes & Unhandled

Reviewer: ```suggestion
## Crashes & Unhandled Errors
```
maybe?

> A _crash_ is when the application had a hard crash, while _unhandled_ is when an error that wasn't explicitly handled by the application. You'll typically be able to view the corresponding issue that captures this event in [sentry.io](https://sentry.io), and errors that did not cause the end of the application should not be included. To search for crash events in **Discover** or on the **Issues** page, filter by `error.unhandled:true`.

Reviewer: ```suggestion
A _crash_ is when the application had a hard crash, while _unhandled_ is an error that wasn't explicitly handled by the application. You'll typically be able to view the corresponding issue that captured the crashed event in [sentry.io](https://sentry.io); errors that did not cause the end of the application should not be included. To search for crash events in **Discover** or on the **Issues** page, filter by `error.unhandled:true`.
```

Reviewer: this paragraph is a bit unclear now that unhandled was added

> Starting with v25.9.0, Sentry differentiates between crashes and unhandled errors and both are counted separately. Check your SDK release notes and update to the latest version to support _unhandled_ session status.

Reviewer: ```suggestion
Starting with v25.9.0, Sentry differentiates between crashes and unhandled errors, and both are counted separately. Check your SDK release notes and update to the latest version to support _unhandled_ session status.
```

The number of users that experienced a crash in the specified time range.

## Sessions vs. Issues

One common misconception is that session data is derived from issues, and that the numbers can be cross referenced. This is not the case.

> Session data is unconditionally sent from SDKs when the integration is enabled (it is enabled by default in most SDKs). This dataset tracks every session as it is started and ended, and includes only basic tags related to release and environment. In contrast, Issues are derived from error events, which are subject to [inbound filters](/concepts/data-management/filtering/) and [sampling](/platform-redirect/?next=/configuration/sampling/) and [quota](/pricing/quotas/) limits.

Reviewer: ```suggestion
Session data is unconditionally sent from SDKs when the integration is enabled (it is enabled by default in most SDKs). This dataset tracks every session start and end, and includes only basic tags related to release and environment. In contrast, issues are derived from error events, which are subject to [inbound filters](/concepts/data-management/filtering/), [sampling](/platform-redirect/?next=/configuration/sampling/), and [quota](/pricing/quotas/) limits.
```

The application timed out, froze, or was forced to quit by the operating system. There is usually no corresponding Sentry issue, as this is a passive action.

### Unhandled

> The application had an explicit error that was not explicitly [unhandled](#crashes--unhandled). The language or framework prevented the process from terminating.

Reviewer: should it say "handled"?
